mne/surface.py

```diff
@@ -32,15 +32,15 @@
 # BEM
 
 @verbose
-def read_bem_surfaces(fname, add_geom=False, s_id=None, verbose=None):
+def read_bem_surfaces(fname, patch_stats=False, s_id=None, verbose=None):
```

Comment:
no… I try to keep up with limited internet access…

I'll let you handle this one then :)